isis/src/osirisrex/tsts/mapcam/Makefile

```diff
@@ -18,15 +18,15 @@ commands:
 	ocams2isis $(TSTARGS)                                                                        \
 	           from=$(INPUT)/20190303T100344S990_map_iofL2pan_V001.fits                          \
 	           to=$(OUTPUT)/20190303T100344S990_map_iofL2pan_V001-spiceinit.cub                  \
-	           >& /dev/null;
+	           ;
```

Comment:
Is there a reason for removing these redirects? Most every makefile test ignores stdout and stderr. Same for other tests below.

Reply:
I only did that to get the error messages coming from Jenkins, since it wasn't reproducible locally. That's how I determined the issue is the image existing pre public release and are not in the new kernels' time range.